components/public-api/gitpod/v1/workspace.proto

```diff
@@ -157,6 +171,44 @@ message StartWorkspaceResponse {
   Workspace workspace = 1;
 }
 
+message GetWorkspaceDefaultImageRequest {
+  // workspace_id specifies the workspace to get default image
+  string workspace_id = 1;
+}
+
+message GetWorkspaceDefaultImageResponse {
+  string default_workspace_image = 1;
+}
+
+message SendHeartBeatRequest {
+  // workspace_id specifies the workspace to send heartbeat
+  //
+  // +required
+  string workspace_id = 1;
+
+  // editor_closed indicates if the editor is closed.
+  // If set to true, the workspace will be stopped after stopping_timeout.
+  bool editor_closed = 2;
```

Comment:
we started using disconneted everywhere, wonder maybe just call it `disconnected`, for ssh editor does not make much sense